spec/litmus_paper/dependency/script_spec.rb

```diff
@@ -8,7 +8,7 @@
     end
 
     it "is false when the script returns 1" do
-      check = LitmusPaper::Dependency::Script.new("false")
+      check = LitmusPaper::Dependency::Script.new("echo Hello && echo Goodbye 1>&2 && false")
```

Comment:
good call out, this was to ensure stdout and stderr are being outputted correctly in the logs